application_sandbox/sample_application_framework/sample_application.h

```diff
@@ -46,7 +47,7 @@ struct SampleOptions {
   bool host_query_reset = false;
   bool extended_swapchain_color_space = false;
   bool shared_presentation = false;
-  bool enable_vulkan_1_1 = false;
+  bool enable_vulkan_1_1 = false;   // This is deprecated, use the |vulkan_api_version| instead for new applications.
```

Comment:
It looks like we can remove this field entirely, since it is only used in `EnableVulkan11` and `buildVulkanApplicationOptions`